src/mame/drivers/savquest.cpp

```diff
@@ -1,5 +1,5 @@
 // license:BSD-3-Clause
-// copyright-holders:R. Belmont, Peter Ferrie
+// copyright-holders:R. Belmont, Peter Ferrie, Nick Lombard
```

Comment:
IMO not enough code contributed to become copyright holder of this MAME driver.

Reply:
Noted

Reply:
Fixed at df8d9ff1